inc/Editor/DataBinding/BlockBindings.php

```diff
@@ -198,6 +198,14 @@ private static function execute_queries( array $block_context, array $source_arg
 		}
 	}
 
+	public static function should_render_empty_result( WP_Block $block ): bool {
+		$block_context = $block->context[ self::$context_name ] ?? [];
+		$query_response = self::execute_queries( $block_context, [], 'remote_data_block_check_for_no_results' );
```

Comment:
need to merge `trunk` I think
```suggestion
		$query_response = self::execute_queries( $block_context, [] );
```